library-udf/src/main/java/org/apache/iotdb/library/dprofile/util/MADSketch.java

```diff
@@ -261,7 +262,8 @@ public double[] getValid_range() {
 
   public void show(Bucket[] buckets) {
     for (Bucket bucket : buckets) {
-      System.out.println(bucket.index + ": " + bucket.count);
+      Logger logger = Logger.getLogger(getClass());
```

Comment:
To make it a final static field is better, you can refer to our other java files for implementation.